frontend/src/js/components/editors/template/TemplateLabelSettings.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+
+
+var TemplateLabelSettings = React.createClass({
+    propTypes: {
+        label: React.PropTypes.object,
+        update: React.PropTypes.func
+    },
+    updateLabelValue(key) {
+        const self = this;
+        let validator = parseFloat;
+        if (key === "priority") {
+            validator = parseInt;
+        }
+        return (e) => {
+            self.props.update(key, validator(this.range(e.target.value)));
+        };
+    },
+    range(alt){
+        if(alt < 0){
+            alt = 0;
+        }
+        else if (alt > 1){
+            alt = 1;
+        }
+        return alt;
+    },
```

Comment:
So this works, and is ugly but we still can't enter a decimal number.

if you type some thing target then one its cast back to one, if you type a decimal point `.` I throws `ValueError: y must be a number`  

so to type `.5` I had to type 1 back arrow add the dot `.1` add the 5 `.15` then remove the 1 `.5` ... thats not right... 

Thoughts?